app/models/protocol.rb

```diff
@@ -288,7 +296,7 @@ def unique_rm_id_to_protocol
     sort_order  = arr[1]
     case sort_name
     when 'id'
-      order("id #{sort_order.upcase}")
+      order("protocols.id #{sort_order.upcase}")
```

Comment:
@kyle-glick @jwiel86 Yeah, It does I think. I had to change this because just 'id' was causing an ambiguity error if the initial query had any eager loading. I'm not sure, but it looks like Hanna's pull request also addresses this with the 'protocols.id'. I think the best thing would be to merge hers in first, then I'll address conflicts with mine, and make sure it still works.